RiotSwiftUI/Modules/UserSessions/Common/View/UserSessionCardViewData.swift

```diff
@@ -37,16 +39,64 @@ struct UserSessionCardViewData {
     /// Indicate if the current user session is shown and to adpat the layout
     let isCurrentSessionDisplayMode: Bool
     
+    /// The name of the shield image to show the verification status.
+    var verificationStatusImageName: String {
+        switch verificationState {
+        case .verified:
+            return Asset.Images.userSessionVerified.name
+        case .unverified:
+            return Asset.Images.userSessionUnverified.name
+        case.unknown:
```

Comment:
A whitespace is missing between `case` and `.unknown`.
And for the three next switches too.

Reply:
Good spot, thanks!